File: thursday/week1/oeyh/2751.py
import sys

n = int(sys.stdin.readline()) #input보단 메모리 사용량이 적음
data = sorted([int(sys.stdin.readline()) for i in range(n)])

#append는 메모리의 재할당이 일어나 속도가 저하되고 메모리 사용량이 올라감

for i in data:
    print(i)
  

File: saturday/week_01/gayion21/2751.js
const input = require("fs").readFileSync('/dev/stdin').toString().trim().split('\n').slice(1).map(num => Number(num));

input.sort((a,b) => a - b);  

console.log(input.join("\n"));

/*
const fs = require("fs");
const input = fs.readFileSync('/dev/stdin').toString().trim().split("\n").map(item => Number(item));

const iter = input.shift();

input.sort((a,b) => a - b);

console.log(input.join("\n"));
*/

/*
let input = []

require('readline')
    .createInterface(process.stdin, process.stdout)
    .on('line', function(x) {
        input.push(x.trim())
    })
    .on('close', function() {
        const [N, ...order] = input.map(Number)
        const result = order.sort((a, b) => a - b)
        console.log(result.join('\n'));
    })
*/